<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sample_4_4Class</class>
 <widget class="QWidget" name="sample_4_4Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sample_4_4</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Slider和QColor</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sliderRed">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="4">
         <widget class="QTextEdit" name="editColor">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>170</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>170</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>240</red>
                <green>240</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sliderGreen">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="sliderBlue">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Alpha</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="sliderAlpha">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>水平</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="sliderH">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="scrollH">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressH">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Dial和LCDNumber</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDial" name="dial">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="digitCount">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>LCD显示进制</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioDec">
             <property name="text">
              <string>十进制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioBin">
             <property name="text">
              <string>二进制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioOct">
             <property name="text">
              <string>八进制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioHex">
             <property name="text">
              <string>十六进制</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>垂直</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSlider" name="sliderV">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScrollBar" name="scrollV">
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QProgressBar" name="progressV">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset resource="sample_4_4.qrc">
         <normaloff>:/images/images/wsl.ico</normaloff>:/images/images/wsl.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sample_4_4.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>sample_4_4Class</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderH</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scrollH</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderH</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressH</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderV</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scrollV</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderV</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressV</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDec</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioHex</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setHexMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBin</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setBinMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOct</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setOctMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
